src/ch/imedias/rsccfx/view/RsccSupportPresenter.java

```diff
@@ -73,8 +66,10 @@ public void initSize(Scene scene) {
   public String validationImage(String token) {
 
     if (validateToken(token)) {
+      view.connectBtn.setDisable(false);
```

Comment:
since both the content of the TextField with the current token and the connectBtn.disableProperty are properties, I think it would be more elegant to have a binding and omit the validateToken method, instead of using setDisable